src/Chapter01/Listing01.22.SampleCILOutput.cs

```diff
@@ -58,7 +58,7 @@ .maxstack 8
     } // End of method System.Void HelloWorld.Program::Main(System.String[])
 
 
-  .method public hidebysig specialname rtspecialname instance void .ctor() cil managed
+  .method public hidebysig specialName rtspecialname instance void .ctor() cil managed
```

Comment:
Sorry I was wrong when I told you to make this change, I did not realize it was an CIL identifier. This should be all lowercase.